light/verifier.go

```diff
@@ -177,6 +177,13 @@ func ValidateTrustLevel(lvl tmmath.Fraction) error {
 	return nil
 }
 
+func ValidateNumSamples(numSamples uint32) error {
+	if numSamples == 0 {
+		return fmt.Errorf("number of samples must be > 0")
+	}
+	return nil
+}
```

Comment:
nit: are we planning on adding more checks in this function? perhaps it would be slightly simpler to just add this check to [the place](https://github.com/lazyledger/lazyledger-core/blob/c97011c66e3341155aea32082d0a162b50e31ab6/light/client.go#L246) where it's needed?

Reply:
> nit: are we planning on adding more checks in this function? 

not sure yet. 

>  perhaps it would be slightly simpler to just add this check to the place where it's needed?

I think the code is more consistent with the already existing code and I would argue that that place is more readable like this (alternatively, we could introduce a variable e.g. `isValidNumSamples := numSamples != 0`. That would the same effect).